shaders/lines_shader.frag

```diff
@@ -0,0 +1,7 @@
+#version 410
+
+out vec4 frag_color;
+
+void main () {
+	frag_color = vec4 (1.0, 0.0, 0.0, 1.0);
+}
```

Comment:
Github is giving a warning about not having a newline at the end of the file. The formatter should add this (I think)

Reply:
Github ? 
I reformatted the file and checked the encoder.